framework/util/to_string.h

```diff
@@ -63,6 +73,9 @@ inline std::string ToString(uint32_t      apiFlags,
     GFXRECON_UNREFERENCED_PARAMETER(toStringFlags);
     GFXRECON_UNREFERENCED_PARAMETER(tabCount);
     GFXRECON_UNREFERENCED_PARAMETER(tabSize);
+    GFXRECON_LOG_ERROR("std::string ToString(uint32_t apiFlags,..) should never be called see: " __FILE__ ", line %d ",
```

Comment:
Could this be LOG_FATAL and then omit the ASSERT?

Reply:
I might have missed it but I don't see `LOG_FATAL` actually killing the process. It just seems to be a verbosity level between _error_ and _always_. 

My intent here is to log under any build, and only kill the process with an assert in debug builds.

I can actually delete all this logging and asserting though if it is a distraction as I see no way this function will ever be called. The log and assert are just sanity checks on that.

Reply:
I went ahead and removed them. The function isn't going to be called without one of use going out of our way to do so.